src/Components/ErrorBoundary.tsx

```diff
@@ -18,7 +28,45 @@ export class ErrorBoundary extends React.Component<Props> {
     }
   }
 
+  static getDerivedStateFromError(error) {
+    /**
+     * Check to see if there's been a network error while asynchronously loading
+     * a dynamic webpack split chunk bundle. Can happen if a user is navigating
+     * between routes and their network connection goes out.
+     *
+     * @see https://reactjs.org/docs/code-splitting.html
+     */
+    if (error.message.match(/Loading chunk .* failed/)) {
+      return {
+        asyncChunkLoadError: true,
+      }
+    }
+  }
+
   render() {
+    if (this.state.asyncChunkLoadError) {
+      return <ErrorModalWithReload show={this.state.asyncChunkLoadError} />
+    }
```

Comment:
Are there other cases we should be checking for? It's not clear to me what this component was really doing previously aside from logging things out.

Reply:
Yup, was just used for logging for the time being. I think we can experiment a bit here!